paseos/visualization_example/utils.py

```diff
@@ -0,0 +1,34 @@
+from paseos import ActorBuilder, SpacecraftActor
+import paseos
+
+import pykep as pk
+
+
+def test_setup():
```

Comment:
explanation for users what this example is

Reply:
Removed.